modules/utxo-lib/test/fixtures_thirdparty/parse.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @prettier
+ */
+import * as assert from 'assert';
+import { getNetworkList, getNetworkName, isBitcoinGold, isMainnet, isZcash } from '../../src/coins';
+import { sigHashTestFile, SigHashTestVector, testFixtureArray, txValidTestFile, TxValidVector } from './fixtures';
+
+const Transaction = require('../../src/transaction');
+
+describe('Third-Party Fixtures', function () {
+  getNetworkList()
+    .filter(isMainnet)
+    .forEach((network) => {
+      describe(`parse ${getNetworkName(network)}`, function () {
+        function roundTripTest(buf: Buffer) {
+          const tx = Transaction.fromBuffer(buf, network);
+          assert.strictEqual(tx.toBuffer().toString('hex'), buf.toString('hex'));
+          assert.strictEqual(tx.clone().toBuffer().toString('hex'), buf.toString('hex'));
+          return tx;
+        }
+
+        function runCheckHashForSignature([
+          rawTransaction,
+          script,
+          inputIndex,
+          hashType,
+          signatureHash,
+        ]: SigHashTestVector) {
+          if (isBitcoinGold(network) && hashType & Transaction.SIGHASH_FORKID) {
```

Comment:
Nit: add parens to make precedence clear here

Reply:
`prettier` strips them off again :(